src/blueprints/tv2/action-factories/tv2-replay-action-factory.ts

```diff
@@ -0,0 +1,236 @@
+import { Action } from '../../../model/entities/action'
+import { Tv2BlueprintConfiguration } from '../value-objects/tv2-blueprint-configuration'
+import { Tv2SourceMappingWithSound } from '../value-objects/tv2-studio-blueprint-configuration'
+import { PartActionType, PieceActionType } from '../../../model/enums/action-type'
+import { PartInterface } from '../../../model/entities/part'
+import { PieceInterface } from '../../../model/entities/piece'
+import { Tv2ActionContentType, Tv2ReplayAction, Tv2ReplayAuxAction } from '../value-objects/tv2-action'
+import { PieceType } from '../../../model/enums/piece-type'
```

Comment:
This file does not exist any more.

Reply:
Hence why it is deleted :) 

The compile error wasn't caugt since we have no test file testing Actions, so they are no being executed for tests. I have added test files for all ActionFactories to avoid that in the future.